x/wasm/relay_pingpong_test.go

```diff
@@ -241,9 +242,9 @@ func (p player) IBCPacketReceive(codeID wasmvm.Checksum, env wasmvmtypes.Env, ms
 
 	if maxVal := store.Get(maxValueKey); maxVal != nil && otherCount > sdk.BigEndianToUint64(maxVal) {
 		errMsg := fmt.Sprintf("max value exceeded: %d got %d", sdk.BigEndianToUint64(maxVal), otherCount)
-		return &wasmvmtypes.IBCReceiveResponse{
+		return &wasmvmtypes.IBCReceiveResult{Ok: &wasmvmtypes.IBCReceiveResponse{
```

Comment:
Would it make sense to have a helper for this constructor?
Could be added in the next wasmvm update if you think it would help

Reply:
ja, would be nice to have constructor methods in wasmvm for ok and err. 👍